src/ui/Facet/Facet.ts

```diff
@@ -1251,7 +1251,7 @@ export class Facet extends Component {
 
   protected updateAppearanceDependingOnState() {
     $$(this.element).toggleClass('coveo-active', this.values.hasSelectedOrExcludedValues());
-    if (!$$(this.element).hasClass('coveo-with-placeholder')) {
+    if (this.nbAvailableValues < 1) {
```

Comment:
Is this check still necessary ?

Isn't just doing 

     $$(this.element).toggleClass('coveo-facet-empty', !this.isAnyValueCurrentlyDisplayed());

all we need to do here ?